html/theme.css

```diff
@@ -0,0 +1,89 @@
+
+
+
+/* I tried to mimic VS Code's Dark+ theme with these colors: */
+
+.hljs-literal,
+.hljs-keyword {
+    color: #569cd6
+}
+
+/* .hljs-type, */
+.hljs-number {
+    color: #b5cea8
+}
+
+.hljs-regexp,
+.hljs-bullet,
+.hljs-string {
+    color: #CE9178
+}
+/* .hljs-subst, */
+.hljs-symbol,
+.hljs-class {
+    color: #4EC9B0
+}
+
+.hljs-built_in,
+.hljs-function {
+    color: #DCDCAA
+}
+/* .hljs-title,
+.hljs-params, */
+.hljs-quote,
+.hljs-comment {
+    color: #6A9955
+}
+/* .hljs-doctag,
+.hljs-meta,
+.hljs-meta-keyword,
+.hljs-meta-string,
+.hljs-code,
+.hljs-emphasis,
+.hljs-strong,
+.hljs-formula, */
+
+.hljs-link {
+    color: #D9E577
+}
+
+
+/* .hljs-selector-id,
+.hljs-selector-class,
+.hljs-selector-attr,
+.hljs-selector-pseudo,
+.hljs-Templat,
+.hljs-template-variable,
+.hljs-deletion,
+.hljs-Reason,
+.hljs-pattern-match,
+.hljs-typing,
+.hljs-constructor,
+.hljs-module-access {
+    color: #000000
+} */
+
+/* .hljs {
+} */
+
+pre {
+    font-family: Consolas, 'Courier New', monospace;
```

Comment:
If the font falls back to `Courier New` (e.g. on macOS or Linux), the code does not look very clear:

`Courier New`:

<img width="652" alt="image" src="https://user-images.githubusercontent.com/4662568/98811760-4ea59e00-245c-11eb-8861-d798dfe4fb11.png">

`monospace`:

<img width="652" alt="image" src="https://user-images.githubusercontent.com/4662568/98811887-844a8700-245c-11eb-940c-da34a842b7cf.png">

Monaco:

<img width="651" alt="image" src="https://user-images.githubusercontent.com/4662568/98811980-aba15400-245c-11eb-94a3-edcbf72a7f89.png">

It looks like we could use the CSS variables to refer to vscode font settings:

```css
code {
    font-family: var(--vscode-editor-font-family);
    font-size: 14px;
}
```

Reply:
I tried the following:

```css
body {
    font-size: var(--vscode-editor-font-size);
}

pre {
    font-family: var(--vscode-editor-font-family);
}

code {
    font-family: var(--vscode-editor-font-family);
}
```

and it looks quite compatible with the editor and when font settings are changed,  the R help webview will automatically adjust too.

Reply:
This look like the best approach to match the style of the help view to the rest of the editor.
AFAIK there are some more variables available to adapt e.g. to the color theme being dark/bright/high-contrast, but no way to access the actual theme colors used for normal syntax highlighting.

Reply:
I tested the dark/light/high-contrast and it already works quite nicely with this added. I guess we could consider this as resolved.